data/free worlds intro.txt

```diff
@@ -751,8 +751,8 @@ mission "FW Katya 2B"
 	clearance
 	passengers 1
 	npc
+		personality nemesis plunder waiting
```

Comment:
?

Reply:
yea I'm not sure why those showed up. I just copied the newest version of the file before I submitted the pull request.